sdk/inc/azure/core/az_mqtt5_rpc_server.h

```diff
@@ -34,11 +34,85 @@
 #define AZ_MQTT5_RPC_QOS 1
 #endif
 
+// ~~~~~~~~~~~~~~~~~~~~ Codec Public API ~~~~~~~~~~~~~~~~~
+
+/**
+ * @brief MQTT5 RPC Server options.
+ *
+ */
+typedef struct
+{
+  /**
+   * @brief timeout in seconds for subscribing
+   */
+  uint32_t subscribe_timeout_in_seconds;
+
+} az_mqtt5_rpc_server_options;
+
 /**
  * @brief The MQTT5 RPC Server.
  *
  */
-typedef struct az_mqtt5_rpc_server az_mqtt5_rpc_server;
+typedef struct az_mqtt5_rpc_server
+{
+  struct
+  {
+    az_span model_id;
+    az_span client_id;
+    az_span command_name;
+    /**
+     * @brief The topic to subscribe to for commands
+     */
+    az_span subscription_topic;
```

Comment:
```suggestion
    az_span commands_subscription_topic;
```

?

Reply:
There can only be one subscription topic per RPC Server, and command is specified within the struct name `az_mqtt5_rpc_server`